### PR TITLE
Send handle to policy for unique secure URLs

### LINKED ISSUE #116: filepicker_image_url doesn't include the handle in the policy
When adding the `policy` and `signature` to the image_url, the policy only includes the `call` and `expiry` keys and no `handle` key which is specified here: https://developers.filepicker.io/docs/security/#createPolicy

It seems like the `handle` key is required to read a secure file

### PR DESCRIPTION
**Pull request by dirkkelly**
Fixes #116 

- Add Database migration to CONTRIBUTING.md
- Add Feature to CHANGELOG.md
- Remove support for Ruby 1.9.3 and Ruby 2.0.0

### CODE REVIEW

**Comment by dirkkelly** [comment]
We may want this to be a configurable option.

File: lib/filepicker_rails/policy.rb [hunk 1]
```diff
@@ -19,9 +19,9 @@ def signature
       OpenSSL::HMAC.hexdigest('sha256', ::Rails.application.config.filepicker_rails.secret_key, policy)
     end
 
-    def self.apply(call = [:read, :convert], keys = ['policy', 'signature'])
+    def self.apply(call: [:read, :convert], keys: ['policy', 'signature'], options: {})
```
Comment (dirkkelly) [inline lib/filepicker_rails/policy.rb]: A change to keyword arguments would break support for 1.9.3, however 1.9.3 has been failing master for months #155

**Comment by dirkkelly** [comment]
@maurogeorge I'm not sure who is maintainers of repo, I notice you have a lot of commits though.

Can you please tell me who to contact about getting a review of this change?

Thanks :)

**Comment by kyuss** [comment]
👍

**Pull request merged**